ansible_rulebook/util.py

```diff
@@ -261,3 +261,21 @@ def _builtin_filter_path(name: str) -> Tuple[bool, str]:
     dirname = os.path.dirname(os.path.realpath(__file__))
     path = os.path.join(dirname, "event_filter", filter_name + ".py")
     return os.path.exists(path), path
+
+
+# TODO(alex): This function should be removed after the
+# controller templates are refactored to deduplicate code
+def process_controller_host_limit(template_obj):
```

Comment:
since it is an util function, better to pass in job_args and control_hosts, and output the composed limit string

Reply:
changed